src/sbl_filing_api/entities/repos/submission_repo.py

```diff
@@ -109,6 +110,13 @@ async def update_submission(submission: SubmissionDAO, incoming_session: AsyncSe
     return await upsert_helper(session, submission, SubmissionDAO)
 
 
+async def expire_submission(submission_id: int):
+    session = SessionLocal()
```

Comment:
shouldn't we use the with context here as well?

Reply:
Yup, updated.